### ISSUE TITLE
Option for a task to ignore weekends

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bardsoftware/ganttproject#760]
```
In some use cases weekends might be used in general but some tasks should 
ignore them. See forum discussion 
https://forum.ganttproject.biz/view/20130312144226_2cfvgln1qmvg2sm2v1azpm8qx#0
```

Original issue reported on code.google.com by `dbarashev` on 13 Mar 2013 at 7:00

**Comment by Bastien-D** [comment]
It should be possible to enforce one specific task during one weekend day with no impact at all on any other task.

Deployment of new IT application version on Sunday to ensure it will not have any impact on customers does not means that all others tasks will also move forward that sunday.

**Comment by Bastien-D** [comment]
I agree that student projects management do not require such things, but real operational life requires it.

**Comment by Kleingeldprinz** [comment]
This would be great. In my Projects, in general there are no packages at weekend. But some tasks had to be done (Like Network component exchanges). These work are only at weekends. But they are not plannable in ganttproject right now.

**Comment by camerondmm** [comment]
This issue also affects projects with resources that are not people. For example, an environmental chamber that needs to run 1000's of hours across many weekends. Not being able to specify weekend work on a task-by-task basis causes significant 'delays' to a project schedule. Maybe this could be solved by allowing specific resources to work on weekends, as opposed to a task based approach?

**Comment by biva** [comment]
Hello, any news on this issue? It would be indeed great to choose for each task if weekends should be included or not. Thanks!

**Comment by FrankyCHAMBERS** [comment]
I think it may be possible to have special calendar per task or resource. For exemple You can have a global project that runs each day in a week, and special tasks that can be done only on monday and wenesday for example, or resource that are available only on tuesday. Other software (projetLibre) allow this.
